src/shared/symbols.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+import Symbol from 'es-symbol'
+
+export const ACTION_HANDLER = Symbol('action creator handler')
```

Comment:
That's a good point. Any ideas for providing access in multiple files?

Will these symbols be equivalent?
```js
// file1.js
const ACTION_HANDLER = Symbol('action creator handler')

// file2.js
const ACTION_HANLER = Symobl('action creator handler')
```

I assumed they will not be, but I think you raise a great point!